pyrasterframes/python/pyrasterframes/rasterfunctions.py

```diff
@@ -0,0 +1,139 @@
+from __future__ import absolute_import
```

Comment:
Add some sort of header comment with a phrase stating the purpose/scope of this file.